index.js

```diff
@@ -50,16 +50,16 @@ class SimpleKeyring extends EventEmitter {
   }
 
   // tx is an instance of the ethereumjs-transaction class.
-  signTransaction (address, tx) {
-    const wallet = this._getWalletForAccount(address)
+  signTransaction (address, tx, opts) {
+    const wallet = this._getWalletForAccount(address, opts)
     var privKey = wallet.getPrivateKey()
     tx.sign(privKey)
     return Promise.resolve(tx)
   }
 
   // For eth_sign, we need to sign arbitrary data:
-  signMessage (withAccount, data) {
-    const wallet = this._getWalletForAccount(withAccount)
+  signMessage (withAccount, data, opts) {
```

Comment:
What happens when `opts` is not provided? Does it work as previously? Is this a breaking change?